web/static/js/stream-management.js

```diff
@@ -0,0 +1,488 @@
+/* On page load, get list of current streams */
+$(function() {
+  var streams = [];
+  $.get("/api/sources", function(data) {
+    $.each(data.sources, function(k, v) {
+      streams[v.id] = v;
+      $("#settings-tab-inputs-stream-selection").append(
+        '<li class="list-group-item list-group-item-action list-group-item-dark stream" style="vertical-align: bottom;" data-id="' + v.id + '">' +
+        v.name +
+        ' <span style="float:right;font-size:0.8rem;color:navy;line-height:25px;vertical-align: bottom;">' + `rca ${v.id+1}` + '</span>'
+      );
+    });
+  });
+
+  $.get("/api/streams", function(data) {
+    $.each(data.streams, function(k, v) {
+      streams[v.id] = v;
+      $("#settings-tab-inputs-stream-selection").append(
+        '<li class="list-group-item list-group-item-action list-group-item-dark stream" style="vertical-align: bottom;" data-id="' + v.id + '">' +
+        v.name +
+        ' <span style="float:right;font-size:0.8rem;color:navy;line-height:25px;vertical-align: bottom;">' + v.type + '</span>'
+      );
+    });
+  });
```

Comment:
It might make sense to combine the requests into one (eventually the zone and group mgmt code would probably be added here as well
```suggestion
  $.get("/api", function(data) {
    $.each(data.sources, function(k, v) {
      streams[v.id] = v;
      $("#settings-tab-inputs-stream-selection").append(
        '<li class="list-group-item list-group-item-action list-group-item-dark stream" style="vertical-align: bottom;" data-id="' + v.id + '">' +
        v.name +
        ' <span style="float:right;font-size:0.8rem;color:navy;line-height:25px;vertical-align: bottom;">' + `rca ${v.id+1}` + '</span>'
      );
    });
    $.each(data.streams, function(k, v) {
      streams[v.id] = v;
      $("#settings-tab-inputs-stream-selection").append(
        '<li class="list-group-item list-group-item-action list-group-item-dark stream" style="vertical-align: bottom;" data-id="' + v.id + '">' +
        v.name +
        ' <span style="float:right;font-size:0.8rem;color:navy;line-height:25px;vertical-align: bottom;">' + v.type + '</span>'
      );
    });
  });
```